src/router_core/connections.c

```diff
@@ -223,8 +224,9 @@ qdr_connection_info_t *qdr_connection_info(bool             is_encrypted,
         pn_data_copy(qdr_conn_properties, connection_properties);
 
     connection_info->connection_properties = qdr_conn_properties;
-    connection_info->tls_ssf = tls_ssf;
-    connection_info->tls     = tls;
+    connection_info->tls_ssf     = tls_ssf;
+    connection_info->tls         = tls;
```

Comment:
I was wondering if we can populate the tls_ordinal and tls_ssf only only if tls is true
Like so - 
```
if (tls) {
       connection_info->tls_ssf = tls_ssf;
       connection_info->tls_ordinal = tls_ordinal;
       connection_info->tls_cipher = strdup(tls_cipher);
       connection_info->tls = tls;
}
```

Just a thought.